components/skus/browser/skus_context_impl.cc

```diff
@@ -44,35 +44,36 @@ logging::LogSeverity GetLogSeverity(skus::TracingLevel level) {
 }  // namespace
 
 namespace skus {
+skus::mojom::SkusResultPtr make_skus_result(const skus::SkusResultCode code,
```

Comment:
this naming convention and underscore casing don't follow chromium conventions. CreateSkusResult and it should be inside `namespace { ... }`

Reply:
Fixed.